providers/AVISO.json

```diff
@@ -123,6 +123,83 @@
         }
     },
     "elevation": {
+        "FES2012": {
+            "format": "FES-netcdf",
+            "constituents": [
```

Comment:
Is this necessary, or will pyTMD auto-guess the constituent names? (previously, I had simply used ` "model_file": "fes2012/data/*_FES2012_SLEV.nc"`, but that wasn't being recognised here)

Reply:
Right, the glob string capability isn't being run for the database as I figured the database will be more "maintained".